src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/compute/AzureComputePluginTest.java

```diff
@@ -222,6 +212,9 @@ public void testRequestInstanceSuccessfully() throws Exception {
         Mockito.doReturn(virtualMachineSize).when(this.azureComputePlugin)
                 .getVirtualMachineSize(Mockito.eq(computeOrder), Mockito.eq(this.azureUser));
 
+        Mockito.doReturn("").when(this.azureComputePlugin)
```

Comment:
You can use the EMPTY_STRING constant of the TestUtil class here.

Reply:
done